sympy/solvers/ode/single.py

```diff
@@ -791,66 +791,93 @@ def _get_general_solution(self, *, simplify: bool = True):
         return [gensol]
 
 
-class SecondNonlinearAutonomousConserved(SinglePatternODESolver):
+class SecondAutonomousNth(SinglePatternODESolver):
     r"""
-    Gives solution for the autonomous second order nonlinear
-    differential equation of the form
+    The general second order autonomous differential equation of the form:
+    .. math :: x''(t) + f(x)*x'(t) + g(x) = 0
 
-    .. math :: f''(x) = g(f(x))
+    This solver is added for the special case in this genre:
+    .. math :: f''(x) = g(f(x)) f'(x)^n
 
-    The solution for this differential equation can be computed
-    by multiplying by `f'(x)` and integrating on both sides,
-    converting it into a first order differential equation.
+    This special case can be solved using by writing the second derivative in
+    form involving a power of x' and on rewriting the second derivative, rearranging.
+
+    For proof of the results, kindly refer the below link:
+    https://en.wikipedia.org/wiki/Autonomous_system_(mathematics)
+    http://eqworld.ipmnet.ru/en/solutions/ode/ode0301.pdf
```

Comment:
This should go in a references section:
https://docs.sympy.org/latest/documentation-style-guide.html

Reply:
Sure, will do.